docs/src/statmodels.md

```diff
@@ -10,8 +10,9 @@ aic
 aicc
 bic
 coef
-coeftable
 confint
+coefnames
```

Comment:
`coefnames` and `coeftable` should come before `confint` AFAIK.

Reply:
Oops. You are right.

Reply:
Still not fixed?